build/Split-Tests.ps1

```diff
@@ -0,0 +1,184 @@
+# ------------------------------------------------------------------------------
+#  Copyright (c) Microsoft Corporation.  All Rights Reserved.
+#  Licensed under the MIT License.  See License in the project root for license information.
+# ------------------------------------------------------------------------------
+
+# This function copies the docs using the moduleMapping.json into their submodule directories.
```

Comment:
Looks like this description was copied and left from the Split-Docs function. Minor, but worth changing to speak regarding tests instead of docs.